subworkflows/nf-core/fastq_align_hisat2/main.nf

```diff
@@ -26,6 +26,10 @@ workflow FASTQ_ALIGN_HISAT2 {
     BAM_SORT_STATS_SAMTOOLS ( HISAT2_ALIGN.out.bam, ch_fasta )
     ch_versions = ch_versions.mix(BAM_SORT_STATS_SAMTOOLS.out.versions)
 
+    publish:
+    HISAT2_ALIGN.out.bam        >> (params.save_align_intermeds ? 'hisat2/' : null)
```

Comment:
Is this going to cause problems/user-confusion having the possibility to publish in two places ( workflow and module, e.g. nf-core modules supplies a path, and then the workflow supplies another path )?

Reply:
The idea is that workflow can overwrite the publish targets defined by processes / subworkflows. And if you can find a convention to agree on, you might not need to overwrite anything in the first place.

But I admit I'm not 100% sold on the `publish:` section for processes. It's more "modular" but if it ends up being overwritten most of the time then it's not very useful.